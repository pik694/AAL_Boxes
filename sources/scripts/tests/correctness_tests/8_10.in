8
3 75 27
80 42 19
30 77 21
87 37 77
19 95 97
19 99 65
1 47 19
49 90 57

