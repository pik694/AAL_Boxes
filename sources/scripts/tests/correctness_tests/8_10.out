6 0 2 3 1 7 5 4 
915901->796583
